### PR TITLE
add a bucket policy to cache bucket

### PR DESCRIPTION
**Pull request by Michenux**
## Description
Add a bucket policy to the cache bucket to improve security.

## Migrations required
NO

## Verification
Runner instance with spot runners.

## Documentation
No impact.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -245,6 +245,32 @@ module "cache" {
   cache_expiration_days                = var.cache_expiration_days
 }
 
+resource "aws_s3_bucket_policy" "cache" {
```
Comment (npalm) [inline main.tf:248]: This policy is already created via the cache module, please check `modules/cache/policies/cache.json`

**Review by npalm — COMMENTED** [review]
Just wondering why you created this PR. the policy is already manged via the cache module.

**Comment by npalm** [comment]
@Michenux can we close this PR?

**Comment by Michenux** [comment]
Yes, u can.

**Pull request closed without merging**